.github/labeler.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
You can also add the `impeller` label to changes under https://github.com/flutter/engine/tree/main/impeller